boards/tsi/tsi/board.yml

```diff
@@ -0,0 +1,10 @@
+board:
+  name: tsi
+  vendor: Tsavorite
+  socs:
+  - name: skyp
+    variants:
+    - name: m85
+      cpucluster: m85
```

Comment:
Was this giving two m85 board types right ? is it fixed ?

Reply:
If we are using only the SoC without variants, it is possible to run with one M85. but, if we want to use the ADSP, we need to provide the necessary configurations with variants.